source/_integrations/microsoft_graph.markdown

```diff
@@ -0,0 +1,76 @@
+---
+title: Microsoft Graph
+description: Instructions on how to query Microsoft's Graph API in Home Assistant.
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: 0.119
+ha_codeowners:
+  - '@jlweston85'
+ha_domain: microsoft_graph
+ha_config_flow: true
+---
+
+The Microsoft Graph integration allows Home Assistant to create sensors from various properties of your Office 365 tenant via Microsoft's Graph API. All queries are performed using the authenticated user's account and depend on the permissions granted when creating the Azure AD application.
+
+- [Configuration](#configuration)
+- [Sensor](#sensor)
+  - [Microsoft Teams](#microsoft-teams)
+
```

Comment:
Integration pages now have automatically a Table of contents, so we don't need to add one manually.

```suggestion
```